### DISCUSSION TITLE
Is it possible to withdraw contributions for an issue?

### DISCUSSION THREAD

**Discussion by forenta** [worknenjoy/gitpay#1078] [Q&A] [answered]
I have some issues with a project that are getting lost in the multitude of existing bugs. I would like to put in a little money to motivate devs or find some new devs for it, but I can't estimate the probability that someone will code a solution. Maybe the developers are overworked, maybe they have other priorities, maybe things are being overlooked.

Hence my question, which I don't yet see solved in the FAQs: Can I take the money I put into the pot for solutions out again after a year if no one has been found to take on solve the issue?

**Comment by alexanmtz** [comment, marked as answer]
Hey @forenta , yeah, you can transfer the bounty to another issue, or now you can create Wallets that you can use across bounties.
